### ISSUE TITLE
Cannot import Manifold from features

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#609]
**Describe the bug**

Cannot import the `Manifold` visualizer from `yellowbrick.features`

**To Reproduce**

```python
from yellowbrick.features import Manifold
```

**Expected behavior**

Should be able to import visualizer from the top level without having to import from `yellowbrick.features.manifold`. 

**Traceback**

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-24-73e973f55599> in <module>()
----> 1 from yellowbrick.features import Manifold

ImportError: cannot import name 'Manifold'

```

**Comment by sivu1** [comment]
`from yellowbrick.features import manifold` works fine. `Manifold` seems to be a typo(caps). isn't it @bbengfort

**Comment by bbengfort** [comment]
@sivu1 the `Manifold` visualizer is a class inside of `yellowbrick.features.manifold`, and it should be directly importable from `yellowbrick.features` as our other visualizers are, e.g. `Rank2D`, `ParallelCoordinates`, etc.

**Comment by bbengfort** [comment]
Fixed in #633
